packages/gestalt/src/ActivationCard.js

```diff
@@ -97,22 +93,30 @@ function ActivationCardLink({ data }: { data: LinkData }): ReactNode {
   );
 }
 
-type CardProps = {
-  ...Props,
-  dismissButton?: {
-    accessibilityLabel: string,
-    onDismiss: () => void,
-  },
-};
+function DismissButton({ onDismiss }: { onDismiss: () => void }): ReactNode {
+  const { accessibilityDismissButtonLabel: accessibilityDismissButtonLabelDefault } = useDefaultLabelContext('ActivationCard');
+  return (
+    <div className={classnames(styles.rtlPos)}>
+      <InternalDismissButton
+        accessibilityLabel={accessibilityDismissButtonLabelDefault}
+        iconColor="gray"
+        onClick={onDismiss}
+        padding={12}
```

Comment:
Previously 
            padding={4}
Now
            padding={12}

Context on this change?

Reply:
Sorry this was part of outdated commit because I hadn't turned on vscode autosave yet. This prop is actually invalid for this component and is now removed.